src/DonorDashboards/resources/js/app/components/subscription-manager/payment-method-control/square-control/index.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import {useImperativeHandle} from 'react';
+import {CreditCard, PaymentForm} from 'react-square-web-payments-sdk';
+
+import './style.scss';
+
+const cardTokenizeResponse = {};
+let cardBrand = 'unknown';
+
+const SquareControl = ({label, value, forwardedRef, gateway}) => {
+
+    const applicationID = gateway.applicationID;
+    const locationID = gateway.locationID;
```

Comment:
These values could be restructured instead of mapping the values one-to-one.

```jsx
const { applicationId, locationId } = gateway;
```

Reply:
Resolved in cef73b95198eb7786eb47a8fe43d29cdaac31ed9